processors/content_extractor.py

```diff
@@ -539,3 +541,49 @@ def extract_utt_content_features_dataloader(cfg, metadata, num_workers):
                 )
                 for index, utt in enumerate(_metadata):
                     extractor.save_feature(utt, batch_content_features[index])
+
+    if cfg.preprocess.extract_phoneme:
```

Comment:
The current code will make SVC and TTS entangle unnecessarily. Move Line545-589 to a new function.